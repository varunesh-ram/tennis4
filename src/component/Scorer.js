import React from 'react';
import { Constants } from '../constants/Constants';

export default class Scorer extends React.Component {
    render() {
        return (
            <div>
                <h5>{Constants.SCORER_HEADER}</h5>
                <label>{Constants.INITIAL_SCORE}</label>
            </div>
        );
    }
}