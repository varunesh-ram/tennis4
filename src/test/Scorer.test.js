import React from 'react';
import { shallow } from 'enzyme';
import Scorer from '../component/Scorer';

describe(("<Scorer/> component"), () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(< Scorer />);
  });

  it("should render correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
  it("should have one heading, label", () => {
    expect(wrapper.find("h5").text()).toEqual("Score");
    expect(wrapper.find("label").length).toBe(1);
  });
});
