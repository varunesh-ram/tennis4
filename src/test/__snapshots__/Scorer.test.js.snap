// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Scorer/> component should render correctly 1`] = `ShallowWrapper {}`;
